### ISSUE TITLE
Cannot create a project

### ISSUE THREAD

**Issue by Fancyflame** [dotnet/runtime#47203]
Problem encountered on https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/install
Operating System: android(ubuntu environment)

When I tried to create a project this was occurred
![Screenshot_2020-12-07-23-49-00](https://user-images.githubusercontent.com/31250625/101372496-ef19a180-38e6-11eb-9222-172113be6704.png)

I have tried to running other softwares on my android device, Rust, Go and C++ work well on the ubuntu environment, so it might be not something wrong with the environment.
Besides, I think my storage is large enough, which has at least 10GB free.
This is the information

![Screenshot_2020-12-07-23-56-25](https://user-images.githubusercontent.com/31250625/101373442-e7a6c800-38e7-11eb-8e6b-dcf8a45b25a1.png)

So is there a way to resolve it? Thanks. :)

**Comment by Fancyflame** [comment]
@kouvel Executing `cat /proc/mounts | grep nfs` returns nothing, `/proc/mounts` is exists and has content in it. `/tmp/.dotnet/shm` is exists but no files in it. I can't tell what to do next. In principle, permission problem won't be occur.

**Comment by kouvel** [comment]
I suspect `flock -e /tmp/.dotnet/shm -c "echo $?"` would show a similar error, not sure where the permissions issue is coming from. On my ubuntu machine the above prints `0` when the user has read permission to the directory. It seems like something is preventing the file lock from being acquired.

**Comment by Fancyflame** [comment]
@kouvel Yes your expectation is right. It thrown a Permission denied error.
